AUTHORS.md

```
@@ -0,0 +1,7 @@
+<!-- SPDX-FileCopyrightText: 2025 The geany contributors.
+     SPDX-License-Identifier: MPL-2.0
+-->
+Authors
+=======
+
+- @AlphaOne1
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)